### PR TITLE
Remove ksonnet references

### PR DESCRIPTION
**Pull request by malcolmholmes**

### CODE REVIEW

File: consul-mixin/README.md [hunk 1]
```diff
@@ -6,16 +6,17 @@ Grafana dashboards and Prometheus alerts for operating Consul, in the form
 of a monitoring mixin. They are easiest to use with the [prometheus-ksonnet](https://github.com/grafana/jsonnet-libs/tree/master/prometheus-ksonnet)
 package.
 
-To install this mixin, use [ksonnet](https://ksonnet.io/):
+To use this mixin, install [Tanka](https://tanka.dev/) and [Jsonnet Bundler](https://github.com/jsonnet-bundler/jsonnet-bundler).
```
Comment (sh0rez) [inline consul-mixin/README.md:9]: More complete jb install instructions are available at https://tanka.dev/install#jsonnet-bundler

File: memcached/README.md [hunk 2]
```diff
@@ -46,11 +36,11 @@ memcached + {
 - Apply your config:
 
 ```
-$ ks apply default
+$ tk apply default
 ```
 # Customising and Extending.
 
-The choice of Ksonnet for configuring these jobs was intention; it allows users
+The choice of Tanka for configuring these jobs was intention; it allows users
```
Comment (sh0rez) [inline memcached/README.md:43]: Shouldn't it be "intentional"?

Comment (malcolmholmes) [reply memcached/README.md:43]: fixed in two places

File: README.md [hunk 3]
```diff
@@ -8,13 +8,13 @@ This repository contains various Jsonnet libraries we use at Grafana Labs:
 * [`grafana-builder`](grafana-builder/): A library for building Grafana dashboards
   with jsonnet, following the builder pattern.
 
-* [`ksonnet-util`](ksonnet-util/): An overlay and set of utilities for [ksonnet](https://ksonnet.io/)
-  that makes working with the library easier.
+* [`ksonnet-util`](ksonnet-util/): An overlay and set of utilities aiming at making
```
[the remark this answers is not in the document]
Comment (malcolmholmes) [reply README.md:11]: Good question. That could go in a kind of "deprecation notice" inside `ksonnet-util`, no? As a separate PR?

**Pull request merged**